sklearn/metrics/tests/test_score_objects.py

```diff
@@ -973,3 +973,29 @@ def test_scorer_no_op_multiclass_select_proba():
         roc_auc_score, needs_proba=True, multi_class="ovo", labels=lr.classes_,
     )
     scorer(lr, X_test, y_test)
+
+
+def test_scorer_general_multiclass():
```

Comment:
It would be good to refactor the `test_classification_scores` as well and have: `test_classification_binary_scores` and `test_classification_multiclass_scores`.

Reply:
I moved the end of ```test_classification_scores``` (the part that tests pickled scorers) to it's own test ```test_custom_scorer_pickling``` and parameterized what remained as ```test_classification_binary_scores``` (also with a couple added scorers).